components/notebook-controller/config/manager/params.env

```diff
@@ -1,5 +1,8 @@
 USE_ISTIO=true
+USE_GATEWAY_API=false
 ISTIO_GATEWAY=kubeflow/kubeflow-gateway
+K8S_GATEWAY_NAME=kubeflow-gateway
+K8S_GATEWAY_NAMESPACE=kubeflow
```

Comment:
I think those need to be included here https://github.com/madmecodes/kubeflow/blob/df73d2217f011642003520c3d16a371beadf8f9e/components/notebook-controller/config/manager/manager.yaml#L24. Did you deploy using the manifests and the new image? Are those propagated?